lib/Perl/MinimumVersion.pm

```diff
@@ -614,6 +614,40 @@ sub _feature_bundle {
 	return (defined($version)?"$version.0":undef, $obj);
 }
 
+my %experimental =
```

Comment:
Shouldn't this use the same hash as feature does?

Reply:
I'm not sure. From the look of it both hashes have slightly different values in some cases.

Reply:
It shouldn't, though it may have a few values that are warnings but not features (such as `regex_sets`) and that hence should be filtered out.

Reply:
Actually, they do differ for `unicode_strings`, but that has good reasons (it was only partially implemented in 5.12) and should probably be ignored here :-)

Reply:
@neilb Should I amend the PR?

Reply:
I suppose you could start with `my %experimental = %features;`, and then override those two values.

Reply:
True. I'm just trying to acertain if there is a chance this PR will ever get merged in, considering it's already over a year old and gathering dust...